nodejs18.x/full-stack/{{cookiecutter.project_name}}/buildspec.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
```

Comment:
I think we don't have such pipeline definitions for other templates since customers can use `sam pipeline` commands. Can we remove this file for that reason?

Reply:
Removed file